experimental/packages/otlp-transformer/package.json

```diff
@@ -59,7 +59,7 @@
     "README.md"
   ],
   "peerDependencies": {
-    "@opentelemetry/api": ">=1.3.0 <1.10.0"
+    "@opentelemetry/api": "^1.3.0"
```

Comment:
This here is fine to unpin as it does not implement the API. :+1:

Reply:
OK, I think I kind of get it now. Took me some time and iterations to really wrap my head around this. I think what confused me additionally is that this package here had this (which, as you stated, does not need it), which is also how we stumbled over this problem in the first place. So I'll just fix this one place instead! 👍